src/resolution.js

```diff
@@ -79,32 +109,31 @@ var ri = module.exports = {
 	* @public
 	*/
 	getScreenType: function (rez) {
-		rez = rez || {
-			height: global.innerHeight,
-			width: global.innerWidth
+		rez = rez || _workspaceBounds || {
+			height: 1080,
+			width: 1920
 		};
-		var i,
-			types = _screenTypes,
-			bestMatch = types[types.length - 1].name,
-			swap;
+
+		var types = _screenTypes,
+			bestMatch = types[types.length - 1].name; // Blindly set the first screen type, in case no matches are found later.
 
 		_orientation = 'landscape';
 
 		if (rez.height > rez.width) {
 			_orientation = 'portrait';
-			swap = rez.width;
+			var swap = rez.width;
 			rez.width = rez.height;
 			rez.height = swap;
 		}
 
-		// loop thorugh resolutions
-		for (i = types.length - 1; i >= 0; i--) {
-			// find the one that matches our current size or is smaller. default to the first.
-			if (rez.width <= types[i].width) {
+		// Loop thorugh resolutions, last->first, largest->smallest
+		for (var i = types.length - 1; i >= 0; i--) {
+			// Find the screenType that matches our current size or is smaller. Default to the first.
```

Comment:
Should this comment say '... matches our current size or is larger'?  I find 'default to first' to be a tad confusing because it's really the last in the list.  Maybe, default to largest?

Reply:
I forgot that I mentioned "first" here. It was a previous iteration of the code. It's supposed to be "last", not "first" now. I can change it to "largest", but "last" is probably more accurate, considering the list is manually sorted, not automatically sorted, so the developer can choose the fallback order.

Reply:
What about the 'larger' comment?  It picks the screen type that is larger than (can contain) the current resolution, not one that is smaller than the current resolution.

Reply:
It's not checking for a larger size, it's checking for a smaller size. This part of the comment or the check didn't change from the previous version.